tests/plugins/autocomplete/autocomplete.js

```diff
@@ -39,18 +39,12 @@
 
 			editor.editable().fire( 'keyup', new CKEDITOR.dom.event( {} ) );
 
-			assertViewOpened( ac, true );
-
-			editable.fire( 'keydown', new CKEDITOR.dom.event( { keyCode: 40 } ) );
-
 			assertViewOpened( ac, true );
 			assert.isTrue( ac.view.getItemById( 1 ).hasClass( 'cke_autocomplete_selected' ) );
```

Comment:
You're asserting that the first item is focused in "test arrow down selects next item". Instead the bug that you have fixed should get a dedicated TC, like "test autocomplete starts with the first item selected".

Same issue happens in the "test arrow up selects previous item" TC, you're testing whether the fist item was focused there too.